src/lib/routes/edge-api/index.ts

```diff
@@ -78,23 +85,24 @@ export default class EdgeController extends Controller {
             middleware: [
                 this.openApiService.validPath({
                     tags: ['Edge'],
+                    summary:
+                        'Accepts batched metrics from Edge, for insertion into our metrics storage',
```

Comment:
This looks like something between a description and a summary (and there is no description).

Something like this?
```suggestion
                    summary:
                        'Send metrics from Edge',
                     description:
                        'This operation accepts batched metrics from Edge. Metrics will be inserted into Unleash's metrics storage',
                     
```

Reply:
Yeah, will update so the description is actually a valid string :)